CHANGELOG.md

```diff
@@ -4,6 +4,12 @@
 
 ## Unreleased
 
+## 0.4.0
```

Comment:
Please change the version to 0.3.2 since the change is minor and does not seem to bring many changes

Reply:
updated